src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+const collum = document.querySelector('tbody').children;
+const tHead = document.querySelector('thead').children[0];
+const tFoot = document.querySelector('tfoot').children[0];
+
+tHead.children[3].insertAdjacentHTML('afterend', `
+<th>${tHead.children[1].textContent}</th>
+`);
+
+tFoot.children[3].insertAdjacentHTML('afterend', `
+<th>${tFoot.children[1].textContent}</th>
+`);
+
+const newCollum = [...collum].map(element => element.children[1].textContent);
```

Comment:
does map mutate initial array?
```suggestion
const newColumn = columns.map(element => element.children[1].textContent);
```